src/aspire/utils/numeric/cupy_fft.py

```diff
@@ -0,0 +1,14 @@
+import cupy as cp
+
+
+class CupyFFT:
+
+    fft = staticmethod(cp.fft.fft)
```

Comment:
Yes. Due to the different default arguments, probably we should not set the default one in the general interface.

Reply:
Last week Vineet and I also discussed that we might use the pyfftw numpy interface instead of the class structure from the pyfftw. This interface has very similar augment lists with scipy.fft and cupy.fft as shown in the webpage,
https://hgomersall.github.io/pyFFTW/pyfftw/interfaces/numpy_fft.html.

This might save some time to uniform our interface.

Reply:
These are still defined as function assignments, not as regular functions using `def`s. Please define regular functions.

> Yes. Due to the different default arguments, probably we should not set the default one in the general interface.

No. We want the default arguments to be *the same* and must therefore enforce this in the general interface. Now the default arguments will depend on the chosen backend, which is problematic.

Reply:
The default arguments are the same starting commit 42fe224.